third_party/amd/lib/TritonAMDGPUToLLVM/SchedInstructions.h

```diff
@@ -0,0 +1,19 @@
+#ifndef TRITON_CONVERSION_TRITONAMDGPU_TO_LLVM_SCHED_INSTRUCTIONS_H
+#define TRITON_CONVERSION_TRITONAMDGPU_TO_LLVM_SCHED_INSTRUCTIONS_H
+
+#include "mlir/IR/Types.h"
+#include "triton/Dialect/Triton/IR/Dialect.h"
+#include "triton/Dialect/TritonGPU/IR/Dialect.h"
+
+namespace mlir::triton {
```

Comment:
Can you add some comments to explain the following APIs in general? Something like "The following functions are used to collect and set side-channel information during to LLVM conversion to facilitate instruction scheduling controls."

Reply:
Done